common/realsense-ui-advanced-mode.h

```diff
@@ -74,6 +74,13 @@ inline bool string_to_int(const std::string& str, float& result)
     }
 }
 
+inline bool is_number(const std::string& str)
```

Comment:
Use c++ built-in std::stol/stoi routines

Reply:
This function encapsulates the usage of the std functions and handles the exceptions for cleaner shorter code relevant place. Should I use this std:stol routines even though I'll have to handle the exception here instead of in a designated function?